packages/amplify-provider-awscloudformation/src/permissions-boundary/permissions-boundary.ts

```diff
@@ -61,7 +61,10 @@ const permissionsBoundarySupplier = async (
 
   const isYes = context?.input?.options?.yes;
   if (required && (isYes || !doPrompt)) {
-    throw new Error('A permissions boundary ARN must be specified using --permissions-boundary');
+    throw new AmplifyError('InputValidationError', {
+      message: 'A permissions boundary ARN must be specified using --permissions-boundary',
+      link: `${AMPLIFY_SUPPORT_DOCS.CLI_PROJECT_TROUBLESHOOTING.url}`,
```

Comment:
this link would be better here: https://docs.amplify.aws/cli/project/permissions-boundary/